A couple of doc tests are failing, just need to bump up the counts:

```
======================================================================
FAIL: ./docs/index.rst
Doctest: index.rst
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/doctest.py", line 2204, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for index.rst
  File "./docs/index.rst", line 0

----------------------------------------------------------------------
File "./docs/index.rst", line 55, in index.rst
Failed example:
    len(axl.strategies)
Expected:
    238
Got:
    239


======================================================================
FAIL: ./docs/how-to/classify_strategies.rst
Doctest: classify_strategies.rst
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/doctest.py", line 2204, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for classify_strategies.rst
  File "./docs/how-to/classify_strategies.rst", line 0

----------------------------------------------------------------------
File "./docs/how-to/classify_strategies.rst", line 81, in classify_strategies.rst
Failed example:
    len(strategies)
Expected:
    55
Got:
    56


----------------------------------------------------------------------
```

--- a/axelrod/tests/strategies/test_grudger.py
+++ b/axelrod/tests/strategies/test_grudger.py
@@ -366,47 +366,74 @@ class TestCapri(TestPlayer):
         # punishment co-players for his/her mistake
         opponent_actions = [C] * 10 + [D] + [C] * 10
         opponent = axl.MockPlayer(actions=opponent_actions)
-        actions = [(C, C)] * 10 + [(C, D)] + [(D, C)]  + [(C,C)] * 9
+        actions = [(C, C)] * 10 + [(C, D)] + [(D, C)] + [(C, C)] * 9
         self.versus_test(opponent, expected_actions=actions)
 
         # never allow the defection more than once
         opponent_actions = [C] * 10 + [D] * 2 + [C] * 10
         opponent = axl.MockPlayer(actions=opponent_actions)
-        actions = [(C, C)] * 10 + [(C, D)] + [(D, D)]  + [(D,C)] * 9
+        actions = [(C, C)] * 10 + [(C, D)] + [(D, D)] + [(D, C)] * 9
         self.versus_test(opponent, expected_actions=actions)
 
     def test_noisy_actions(self):
         # accept punishment when making a mistake
-        actions1 = [C,C,D,C,C,C,C]
-        actions2 = [C,C,C,D,C,C,C]
+        actions1 = [C, C, D, C, C, C, C]
+        actions2 = [C, C, C, D, C, C, C]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0.1, seed=20)
+        self.versus_test(
+            opponent,
+            expected_actions=list(zip(actions1, actions2)),
+            noise=0.1,
+            seed=20,
+        )
 
         # recover the cooperation when the opponent cooperated from mutual defection
-        actions1 = [C,D,D,D,D,C,C,C]
-        actions2 = [D,D,D,D,C,C,C,C]
+        actions1 = [C, D, D, D, D, C, C, C]
+        actions2 = [D, D, D, D, C, C, C, C]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0)
+        self.versus_test(
+            opponent, expected_actions=list(zip(actions1, actions2)), noise=0
+        )
 
         # recover the cooperation when the player cooperated by mistake from mutual defection
-        actions1 = [C,D,D,D,C,C,C,C]
-        actions2 = [D,D,D,D,D,C,C,C]
+        actions1 = [C, D, D, D, C, C, C, C]
+        actions2 = [D, D, D, D, D, C, C, C]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0.1, seed=72)
+        self.versus_test(
+            opponent,
+            expected_actions=list(zip(actions1, actions2)),
+            noise=0.1,
+            seed=72,
+        )
 
         # recover the cooperation when the focal and the opponent player cooperated by mistake from mutual defection
-        actions1 = [C,D,D,D,C,C,C,C]
-        actions2 = [D,D,D,D,C,C,C,C]
+        actions1 = [C, D, D, D, C, C, C, C]
+        actions2 = [D, D, D, D, C, C, C, C]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0.1, seed=72)
+        self.versus_test(
+            opponent,
+            expected_actions=list(zip(actions1, actions2)),
+            noise=0.1,
+            seed=72,
+        )
 
         # in other cases, defect
-        actions1 = [C,D,C,C,D,D,D,D]
-        actions2 = [D,D,D,D,D,C,C,D]
+        actions1 = [C, D, C, C, D, D, D, D]
+        actions2 = [D, D, D, D, D, C, C, D]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0.1, seed=452)
+        self.versus_test(
+            opponent,
+            expected_actions=list(zip(actions1, actions2)),
+            noise=0.1,
+            seed=452,
+        )
 
-        actions1 = [C,D,C,D]
-        actions2 = [C,C,C,D]
+        actions1 = [C, D, C, D]
+        actions2 = [C, C, C, D]
         opponent = axl.MockPlayer(actions=actions2)
-        self.versus_test(opponent, expected_actions=list(zip(actions1,actions2)), noise=0.1, seed=15)
+        self.versus_test(
+            opponent,
+            expected_actions=list(zip(actions1, actions2)),
+            noise=0.1,
+            seed=15,
+        )

--- a/docs/how-to/classify_strategies.rst
+++ b/docs/how-to/classify_strategies.rst
@@ -79,7 +79,7 @@ range of memory_depth values, we can use the 'min_memory_depth' and
     ... }
     >>> strategies = axl.filtered_strategies(filterset)
     >>> len(strategies)
-    55
+    56
 
 We can also identify strategies that make use of particular properties of the
 tournament. For example, here is the number of strategies that  make use of the

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -53,7 +53,7 @@ Count the number of available players::
 
     >>> import axelrod as axl
     >>> len(axl.strategies)
-    238
+    239
 
 Create matches between two players::
 

